Mathlib/CategoryTheory/Presentable/CardinalFilteredPresentation.lean

```diff
@@ -115,6 +116,17 @@ lemma presentable [LocallySmall.{w} C] (X : C) : IsPresentable.{w} X := by
   have := (h.colimitPresentation X).isCardinalPresentable κ (by infer_instance) κ' le hκ'
   exact isPresentable_of_isCardinalPresentable _ κ'
 
+-- TODO: Move `AreCardinalFilteredGenerators` to the `ObjectProperty` namespace
+include h in
+lemma isStrongGenerator :
+    ObjectProperty.IsStrongGenerator (Set.range G : ObjectProperty C) :=
+  .mk_of_exists_colimitsOfShape (fun X ↦ ⟨h.J X, inferInstance, by
+    change ObjectProperty.colimitsOfShape (Set.range G) (h.J X) X
+    rw [← ObjectProperty.colimitsOfShape_isoClosure]
+    refine ⟨h.colimitPresentation X, fun j ↦ ?_⟩
+    obtain ⟨i, ⟨e⟩⟩ := h.exists_colimitPresentation_diag_obj_iso X j
+    exact ⟨_, ⟨i, rfl⟩, ⟨e⟩⟩⟩)
```

Comment:
You'll certainly know this better than me, but I think `ObjectProperty.ofObj` is the intended way of turning a family into an `ObjectProperty`, instead of coercing `Set.range` (implicitly abusing the defeq `ObjectProperty C = Set C`.). It also looks like the proof can go through without using `change`.

```suggestion
lemma isStrongGenerator :
    ObjectProperty.IsStrongGenerator (.ofObj G) :=
  .mk_of_exists_colimitsOfShape (fun X ↦ ⟨h.J X, inferInstance, by
    rw [← ObjectProperty.colimitsOfShape_isoClosure]
    refine ⟨h.colimitPresentation X, fun j ↦ ?_⟩
    obtain ⟨i, e⟩ := h.exists_colimitPresentation_diag_obj_iso X j
    exact ⟨_, .mk i, e⟩⟩)
```

Reply:
Yes, this is now possible following the refactor #30269